src/nvim/popupmenu.c

```diff
@@ -611,6 +612,8 @@ void pum_redraw(void)
 
   grid_assign_handle(&pum_grid);
 
+  pum_grid.mouse_enabled = true;
```

Comment:
Not clear why this is done in the middle of `pum_redraw`. Should it be part of grid initialization? I see that you also added it to `win_alloc`.